lib/services.identity.js

```diff
@@ -162,6 +169,18 @@ bedrock.events.on('bedrock-express.configure.routes', function(app) {
     ensureAuthenticated,
     validate({query: 'services.identity.getIdentitiesQuery'}),
     function(req, res, next) {
```

Comment:
This API call has documentation below it that does not apply at all. This should be commented out or updated in some way.